charts/piraeus/templates/operator-csi-driver.yaml

```diff
@@ -4,6 +4,16 @@ kind: LinstorCSIDriver
 metadata:
   name: {{ template "operator.fullname" . }}
   namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ template "operator.fullname" . }}-csi-driver
+{{- with .Values.csi.customLabels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+  annotations:
+    app.kubernetes.io/name: {{ template "operator.fullname" . }}-csi-driver
```

Comment:
same as before. should be matching the resource name and remove the name from annotations.

(Yes, I know our naming is a bit messed up and not perfectly logical, but I think having it consistent is better than only fixing it half-way)